Makefile

```diff
@@ -18,6 +18,7 @@ GOARCH := $(if $(GOARCH),$(GOARCH),amd64)
 GOENV  := GO15VENDOREXPERIMENT="1" CGO_ENABLED=0 GOOS=$(GOOS) GOARCH=$(GOARCH)
 GO     := $(GOENV) go build -trimpath
 
+IMAGE_TAG ?= latest
```

Comment:
IMAGE_TAG is added to change image tag because in e2e we use git hash to avoid potential conflicts